addons/l10n_it_edi/models/res_company.py

```diff
@@ -115,3 +115,10 @@ def _check_tax_representative(self):
     def _compute_l10n_it_edi_proxy_user_id(self):
         for company in self:
             company.l10n_it_edi_proxy_user_id = company.account_edi_proxy_client_ids.filtered(lambda x: x.proxy_type == 'l10n_it_edi')
+
+
+    @api.onchange('l10n_it_has_tax_representative')
+    def _onchange_l10n_it_has_tax_representative(self):
+        for company in self:
```

Comment:
Why not a compute instead?

Reply:
Still hanging

Reply:
What? Absolutely not, a compute would work 100% (store=True as it's a stored field)

Reply:
Hello @JulienAlardot, 

I moved to using the compute and including and inverse. Please see my recent changes when you have the chance.